### ISSUE TITLE
Compression when writing bools with parquet

### ISSUE THREAD

**Issue by pierce314159** [Bears-R-Us/arkouda#2579]
While working #2539, it seems like we don't have compression support for writing boolean arrays with parquet. We should look into if that is supported by `pyarrow` and if it is we should figure out how to add it

**Comment by Ethan-DeBandi99** [comment]
I verified the `pyarrow` does support writing `bool` values with compression. By default, `pyarrow` uses the `RLE` encoding. This is what Arkouda uses as well. 

The specific error we are receiving is 
```
RuntimeError: problem writing to file 1 errors: Error: ParquetError 457 coforall_fn:ParquetMsg Not yet implemented: Selected encoding is not supported.
```

**Comment by Ethan-DeBandi99** [comment]
I was able to get the write to work properly by removing the use of RLE encoding. However, this results in a read error.

**Comment by Ethan-DeBandi99** [comment]
One thing to note is that we may have to remove RLE compression altogether for the multi-column case. I am going to look into ways to avoid this if possible. However, it does appear that if no encoding is explicitly set, it will be inferred. We may want to just switch to this in all cases.

**Comment by Ethan-DeBandi99** [comment]
I ran a bunch of tests this morning to confirm that if no specific encoding is specified Parquet will automatically select the "best" encoding. I was able to verify this as file sizes when specifying an encoding are identical to those when the encoding is not specified. As a result, I believe that it will be best to remove the specification in our code to use `RLE` as this is what is causing the issue for `bool` values. The only difference in file sizes comes with the compression.
